detection-rules/shopify_infra_abuse.yml

```diff
@@ -0,0 +1,35 @@
+name: "Shopify infrastructure abuse"
+description: "Attackers have been observed using myshopify.com links to bypass domain reputation checks."
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and any(body.links,
+          .href_url.domain.root_domain == "myshopify.com"
+  
+          // negate legitimate Shopify newsletters
```

Comment:
should this section be within the `any(body.links, ...` or on it's own outside of that loop?

Reply:
Re-reading this months later, I'm not sure what was going on here. I've restructured it to make a lot more sense.